submissions/jeff-jeremy/scripts/controllers/articleController.js

```diff
@@ -19,6 +21,8 @@
 
   // COMMENT: What does this method do?  What is it's execution path?
   articlesController.loadByAuthor = function(ctx, next) {
+    //This method performs a SQL query that selects a specified author within the authors column and returns
+    // that articles author and sets the article(s) with that author to ctx.articles.
```

Comment:
And what is the execution path? How does this method get called, and where does it go afterwards?